src/scene_title.h

```diff
@@ -103,6 +102,9 @@ class Scene_Title : public Scene {
 	/** Contains the state of continue button. */
 	bool continue_enabled;
 
+	/** If it returns to the scene from Scene_Load, check this to do a quick transition */
+	bool quick_transition;
+
```

Comment:
I really don't like this commit.
It is very invasive considering the only purpose is a minor problem which is restarting the BGM when going from load to title or did I miss something else?

Reply:
It's not only the BGM, it's not cleaning the cache of all the data and restart the Scene_Title from the beginning, and also, make sure that the TransitionIn from Title->Load to Title is a simple quick one, instead of the slow FadeIn. The only way I found for Scene_Title to detect this case was using a new "bool" (we cannot reuse "restart_title_cache").